src/styles/components/widgets/_dropdown.scss

```diff
@@ -1,127 +1,120 @@
 .dropdown-container {
-  @import 'node_modules/react-select/scss/default';
   flex-grow: 1;
 
   &:not(:last-child) {
     margin-bottom: var(--spacing-quarter-unit);
   }
 
   .Select {
-    &.is-focused {
-      & > .Select-control {
-        background-color: var(--color-background-inputs);
-        .Select-input {
-          background-color: var(--color-background-inputs) !important;
-        }
-      }
-    }
-    &.is-open {
-      & > .Select-control {
+    &__control {
```

Comment:
aie, I think there's going to be streambed css weirdness in this one... 
I think I remember having to style some of this in a special way, we'll see..

Reply:
I expect to remove a bunch of "integration" stuff in streambed after this, as it won't be necessary anymore. Especially multi-select related stuff. Won't be a trivial update.